pkg/adaptor/cloud/aws/manager.go

```diff
@@ -36,7 +36,7 @@ func (_ *Manager) ParseCmd(flags *flag.FlagSet) {
 	flags.IntVar(&awscfg.RootVolumeSize, "root-volume-size", 30, "Root volume size (in GiB) for the Pod VMs")
 	// Setting disable-cvm to true as there are still some rough edges with CVMs.
```

Comment:
I guess we should also remove the comment?

Reply:
Ah yes. Thanks for spotting it